google/auth/_default.py

```diff
@@ -285,7 +309,7 @@ def default():
         _get_explicit_environ_credentials,
         _get_gcloud_sdk_credentials,
         _get_gae_credentials,
-        _get_gce_credentials)
+        lambda: _get_gce_credentials(request))
```

Comment:
Prefer `functools.partial(_get_gce_credentials, request)` over a `lambda` (I try to never use `lambda` if I can help it).